test/models/work_test.rb

```diff
@@ -0,0 +1,86 @@
+require "test_helper"
+
+describe Work do
+  let(:work2) { works(:one) }
+  let(:user) { users(:one) }
+  let(:work) { 
+    Work.new(category: "book", title: 'test book', creator: "author", publication_year: 2019) 
+  }
+
+  describe "validations" do
+    it "must be valid with good data" do
+      value(work).must_be :valid?
+    end
+
+    it "fails without a category" do
+      work.category = nil
+
+      result = work.valid?
+
+      expect(result).must_equal false
+      expect(work.errors.messages).must_include :category
+    end
+
+    it "fails without a title" do
+      work.title = nil
+
+      result = work.valid?
+
+      expect(result).must_equal false
+      expect(work.errors.messages).must_include :title
+    end
+
+    it "fails without a creator" do
+      work.creator = nil
+
+      result = work.valid?
+
+      expect(result).must_equal false
+      expect(work.errors.messages).must_include :creator
+    end
+
+    it "fails without a publication year" do
+      work.publication_year = nil
+
+      result = work.valid?
+
+      expect(result).must_equal false
+      expect(work.errors.messages).must_include :publication_year
+    end
+
+    it "fails if the publication year isn't an integer" do
+      work.publication_year = "year"
+
+      result = work.valid?
+
+      expect(result).must_equal false
+      expect(work.errors.messages).must_include :publication_year
+    end
+  end
+
+  describe "media spotlight" do
+    it "can return a work that was voted most" do
+      perform_vote(work_id: work2.id, user_id: user.id)
+
```

Comment:
For this and the `top_ten` method, what happens if there are no works? What happens if there are works but no votes?